lib/chewy/type/witchcraft.rb

```diff
@@ -223,7 +223,7 @@ def replace_lvar(node, old_variable, new_variable)
         def replace_send(node, variable)
           if node.is_a?(Parser::AST::Node)
             if node.type == :send && node.children[0].nil?
-              node.updated(nil, [Parser::AST::Node.new(:lvar, [variable]), *node.children[1..-1]])
+              node.updated(nil, [Parser::AST::Node.new(:lvar, [variable]), *node.children[1..]])
```

Comment:
I am not really sure that ruby 2.5 is compatible with this syntax.
Since we say at [this part of our README](https://github.com/toptal/chewy#ruby) that we support Ruby 2.5, I guess we can't apply this.

Reply:
As Ivan said, the thing that is not right is the README, not the check... So fix the README and forget about this comment.